lib/matplotlib/testing/decorators.py

```diff
@@ -342,52 +357,16 @@ def _image_directories(func):
     """
     Compute the baseline and result image directories for testing *func*.
     Create the result directory if it doesn't exist.
+
+    If the test function is called `test_foobar` and lives in the file
+    /path/to/test_dir/test_script.py then the baseline directory will
+    be /path/to/test_dir/baseline_images/test_script/ and the result
+    directory will be /path/to/test_dir/result_images/test_script/.
```

Comment:
This isn't quite right.  The `result_images` dir is relative to where the tests are run from, not `/path/to/test_dir`.  (The idea is that the test runner shouldn't write any files in the source tree).  It doesn't like the code actually changes this in that regard, I think the docstring is just incorrect here.